src/shared/UniversalRender.jsx

```diff
@@ -156,6 +153,7 @@ async function universalRender({location, initial_state, offchain, ErrorPage, ta
 
     if (process.env.BROWSER) {
         const store = createStore(rootReducer, initial_state, middleware);
+        // setStore(store);
```

Comment:
setStore is not being used by universalRender; what's going on here?

Reply:
I will enable setStore later